src/app/api/reference-data/api/referenceData.service.ts

```diff
@@ -445,4 +445,38 @@ export class ReferenceDataService {
       }
     );
   }
+
+  referenceResourceTypeResourceKeyPost(
```

Comment:
would be good to update the docs [here](https://github.com/dvsa/cvs-svc-reference-data/blob/develop/docs/reference-data-api.yml) and generate the service from those docs

Reply:
Emily will take over this to add editing and removing functionality, it should be implemented then.